docs/shortcuts.html

```diff
@@ -1,664 +1,837 @@
-              <td> <span class="kbd">Opt</span> + <span class="kbd">Click on arrow icon</span> </td>
-            </tr>
-            <tr>
-              <td> Edit attribute </td>
-              <td> <span class="kbd">Enter</span>, <span class="kbd">Double-click on attribute</span> </td>
-              <td> <span class="kbd">Enter</span>, <span class="kbd">Double-click on attribute</span> </td>
-            </tr>
-            <tr>
-              <td> Hide element                   </td>
-              <td> <span class="kbd">H</span> </td>
-              <td> <span class="kbd">H</span> </td>
-            </tr>
-            <tr>
-              <td> Toggle edit as HTML            </td>
-              <td> <span class="kbd">F2</span> </td>
-              <td>  </td>
-            </tr>
-          </tbody>
-        </table>
-
-        <p> Right-clicking an element you can: </p>
-
-        <ul>
-          <li> Force element pseudo states: (<code>:active</code>, <code>:hover</code>, <code>:focus</code>, <code>:visited</code>) </li>
-          <li> Set breakpoints on the elements: (Subtree modifications, Attribute modification, Node removal) </li>
-          <li>Clear console</li>
-        </ul>
+  <h2 id="styles-sidebar"> Styles Sidebar </h2>
+
+  <table>
+    <tbody>
+      <tr>
+        <th></th>
+        <th> Windows / Linux </th>
+        <th> Mac </th>
+      </tr>
+      <tr>
+        <td> Edit rule </td>
+        <td> <span class="kbd">Single-click</span> </td>
+        <td> <span class="kbd">Single-click</span> </td>
+      </tr>
+      <tr>
+        <td> Insert new property </td>
+        <td> <span class="kbd">Single-click on whitespace</span> </td>
+        <td> <span class="kbd">Single-click on whitespace</span> </td>
+      </tr>
+      <tr>
+        <td> Go to line of style rule property declaration in source </td>
+        <td> <span class="kbd">Ctrl</span> + <span class="kbd">Click on property</span> </td>
+        <td> <span class="kbd">Cmd</span> + <span class="kbd">Click on property</span> </td>
+      </tr>
+      <tr>
+        <td> Go to line of property value declaration in source </td>
+        <td> <span class="kbd">Ctrl</span> + <span class="kbd">Click on property value</span> </td>
+        <td> <span class="kbd">Cmd</span> + <span class="kbd">Click on property value</span> </td>
+      </tr>
+      <tr>
+        <td> Cycle through the color definition value </td>
+        <td> <span class="kbd">Shift</span> + <span class="kbd">Click on color picker box</span> </td>
+        <td> <span class="kbd">Shift</span> + <span class="kbd">Click on color picker box</span> </td>
+      </tr>
+      <tr>
+        <td> View auto-complete suggestions </td>
+        <td> <span class="kbd">Ctrl</span> + <span class="kbd">Space</span> </td>
+        <td> <span class="kbd">Cmd</span> + <span class="kbd">Space</span> </td>
```

Comment:
it probably was, it looks like it shows autocomplete suggestions by default now without having to do anything